templates/experiments/experiment_form.html

```diff
@@ -81,6 +82,28 @@ <h3 class="text-md font-bold">{% translate "Please note" %} <i class="fas fa-exc
         {% render_form_fields form "tools" %}
       </div>
 
+      <input type="radio" name="main_tabs" role="tab" class="tab" aria-label="Allowlist">
+      <div role="tabpanel" class="tab-content">
+        <h3 class="text-md font-bold">{% translate "Please note" %}</h3>
+        <p class="py-4 text-sm">
+          {% blocktranslate %}
+            To limit the set of particiapants that can chat with this bot, simply add their identifiers in this list.
+            When the list is empty, the bot will accessible via its public link.
+          {% endblocktranslate %}
+          <div class="w-96 my-2" x-data="{allowlist: {{ form.initial.participant_allowlist }}}">
```

Comment:
Why does this need the alpine attrs?

Reply:
It's mainly for the the copy feature. the copyToClipboard method wants to copy the contents of the div that we point it to, but if we point it to the select div (assuming we're not using the hidden input and populating its contents with alpineJS), then it copies the `<option>` tags as well. It basically copies the HTML. Having a hidden input there solves this issue, but also serves as a convenient way to store the data to be submitted. Hope this makes sense?